test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -76,10 +76,15 @@ Describe "Get-ChildItem" -Tags "CI" {
         }
 
         It "Should return items recursively when using 'Include' or 'Exclude' parameters" {
-            (Get-ChildItem -Path $TestDrive -Depth 1).Count | Should Be 6
+            (Get-ChildItem -Path $TestDrive -Depth 1).Count | Should Be 6 
             (Get-ChildItem -Path $TestDrive -Depth 1 -Include $item_G).Count | Should Be 1
             (Get-ChildItem -Path $TestDrive -Depth 1 -Exclude $item_a).Count | Should Be 5
         }
+        
+        It "get-childitem -LiteralPath should accept 'Include' or 'Exclude' filter" {
+            (Get-ChildItem -LiteralPath $PSHOME -Exclude *).Count | Should Be 0
+            (Get-ChildItem -LiteralPath $PSHOME -Include *.dll).Count | Should Be (Get-ChildItem $PSHOME -Include *.dll).Count
```

Comment:
I'd prefer to use a directory tree created in `BeforeAll`. We can use previous test if replece Path with LiteralPath.

Reply:
Good point, but I also propose we are using $pshome as it is complicated enough. Many customer feedback bugs are failing on this exact scenario. Besides, this just helps me find another issue, it works~

Reply:
I expect a symmetric in tests. Previous test use `-Path`. If we replace `-Path` with '-LiteralPath' we get symmetric test. This makes it easy to understand these tests. The test with `$PSHOME` could be additional test. But what do it really test? What is difference? If it test another code path(s) it is better to be explicit and create a test directory tree for these cases - $PSHOME doesn't make it clear which code path we're really checking.